src/module_wrap.h

```diff
@@ -50,6 +50,7 @@ class ModuleWrap : public BaseObject {
   v8::Persistent<v8::String> url_;
   bool linked_ = false;
   std::unordered_map<std::string, v8::Persistent<v8::Promise>> resolve_cache_;
+  v8::Local<v8::Context> context_;
```

Comment:
Must be a Persistent, Locals are only valid for the duration of their enclosing HandleScope.

Reply:
thank you!! i completely overlooked this 😄